src/commands/run.rs

```diff
@@ -239,6 +239,20 @@ fn enable_plugins(topology: &Topology, cluster_dir: &Path, picodata_path: &Path)
     Ok(())
 }
 
+fn get_port_from_liquid_var(env_vars: &BTreeMap<String, String>, variable: &str) -> Option<u16> {
+    if let Some(env_http_port) = env_vars.get(variable) {
+        return Some(
+            env_http_port
+                .split(':')
+                .nth(1)
+                .unwrap_or_else(|| panic!("could not split {variable} to port and address"))
+                .parse()
+                .unwrap_or_else(|e| panic!("could not parse the port of {variable}: {e}")),
+        );
```

Comment:
По идее, можем воспользоваться встроенным парсером IPv4?

```suggestion
        let addr: SocketAddrV4 = env_http_port
            .parse()
            .unwrap_or_else(|e| panic!("could not parse ip:port from {variable}: {e}"));

        return Some(addr.port());
```